sdk/spring/azure-spring-boot-samples/azure-spring-cloud-sample-eventhubs-binder/README.md

```diff
@@ -111,7 +111,7 @@ Please follow [create managed identity][create-managed-identity] to set up manag
 
 ##### Update MSI related properties
 
-1.  Update [application-mi.yaml](src/main/resources/application-mi.yaml)
+1.  Update [application-mi.yaml][application-mi.yaml]
```

Comment:
nit: if the text is the same as the reference link text, we could use only `[application-mi.yaml]`.